Azure/azure-cli
# [Keyvault] Fix #16390: `az keyvault set-policy`: Allow clear permissions

**Related command**

`az keyvault set-policy`

**Description**

Fix #16390 

**Testing Guide**

`az keyvault set-policy -n $vault --object-id $userObjectId --certificate-permissions` to clear all certificate permissions


---

This checklist is used to make sure that common guidelines for a pull request are followed.
